docs/guides/converting_and_upgrading/orbax_upgrade_guide.rst

```diff
@@ -179,6 +179,34 @@ Then, you can call ``orbax.checkpoint.AsyncCheckpointer.wait_until_finished()``
 
 For more details, read the `checkpoint guide <https://flax.readthedocs.io/en/latest/guides/training_techniques/use_checkpointing.html#asynchronized-checkpointing>`_.
 
+You can also use Orbax AsyncCheckpointer with Flax APIs through async manager. Async manager internally calls wait_until_finished(). This solution is not actively mainted and the recommedation is to use Orbax async checkpointing.
+
+For example:
+
+.. codediff::
+  :title: flax.checkpoints(async), orbax.checkpoint(async)
+  :skip_test: flax.checkpoints
+  :sync:
+
+  PURE_CKPT_DIR = '/tmp/orbax_upgrade/pure'
+  flax.config.update('flax_use_orbax_checkpointing', True)
+  async_manager = checkpoints.AsyncManager()
+
+  checkpoints.save_checkpoint(PURE_CKPT_DIR, CKPT_PYTREE, step=0, overwrite=True, async_manager=async_manager)
+  checkpoints.restore_checkpoint(PURE_CKPT_DIR, target=TARGET_PYTREE)
+  ---
+
+  PURE_CKPT_DIR = '/tmp/orbax_upgrade/pure'
+
+  ckptr = orbax.checkpoint.AsyncCheckpointer(orbax.checkpoint.PyTreeCheckpointHandler())  # A stateless object, can be created on the fly.
```

Comment:
Let's update the APIs from the beginning. 

```
import orbax.checkpoint as ocp
ckptr = ocp.AsyncCheckpointer(ocp.StandardCheckpointHandler())
ckptr.save(dir, args=ocp.args.StandardSave(pytree))
ckptr.wait_until_finished()
ckptr.restore(dir, args=ocp.args.StandardRestore(target))
```

Reply:
Updated.